In [2]:
from helpers.environment import Environment, EnvironmentType
from helpers.device import Device, Motor, AirConditioned, Light, LightType
from helpers.installation import Installation, InstallationType


environments = [
	Environment(
		"Escritório 1",
		2.30,
		3.9,
		EnvironmentType.OFFICES_AND_CLASSROOMS,
		Light(15, LightType.LED),
		[AirConditioned("Ar condicionado escritório 1", 2000)]
	),

	Environment(
		"Escritório 2",
		2.70,
		5.10,
		EnvironmentType.OFFICES_AND_CLASSROOMS,
		Light(15, LightType.LED),
		[AirConditioned("Ar condicionado escritório 2", 2000)]
	),

	Environment(
		"Recepção",
		2.80,
		5.10,
		EnvironmentType.WAITING_ROOMS_AND_COMMUNAL_AREAS,
		Light(15, LightType.LED),
		[AirConditioned("Ar condicionado recepção", 2000)]
	),

	Environment(
		"Vestiário",
		2.25,
		2.70,
		EnvironmentType.BATHROOMS_AND_OTHER_FACILITIES,
		Light(15, LightType.LED),
		[]
	),

	Environment(
		"Cozinha",
		2.70,
		1.90,
		EnvironmentType.KITCHENS_AND_DINING_ROOMS,
		Light(15, LightType.LED),
		[Device("Chaleira", 1200, 1.0)]
	),

	Environment(
		"Refeitório",
		2.70,
		2.90,
		EnvironmentType.RESTAURANTS_CAFETERIAS_BARS_AND_HOTELS,
		Light(15, LightType.LED),
		[Device("Microondas", 2000, 0.92)]
	),

	Environment(
		"Depósito",
		2.70,
		1.30,
		EnvironmentType.WAREHOUSES_DEPOSITS_AND_STORAGE_AREAS,
		Light(15, LightType.LED),
		[]
	),

	Environment(
		"Banheiro",
		1.95,
		1.2,
		EnvironmentType.BATHROOMS_AND_OTHER_FACILITIES,
		Light(15, LightType.LED),
		[Device("Chapinha / Secador", 1200, 1.0)]
	),

	Environment(
		"Banho",
		1.99562,
		1.99562,
		EnvironmentType.BATHROOMS_AND_OTHER_FACILITIES,
		Light(15, LightType.LED),
		[Device("Chuveiro", 7000, 1.0), Device("Chapinha / Secador", 1200, 1.0)]
	),

	Environment(
		"Supervisão de Produção",
		3.28633,
		3.28633,
		EnvironmentType.OFFICES_AND_CLASSROOMS,
		Light(15, LightType.LED),
		[]
	),

	Environment(
		"Produção 1",
		7.5,
		8,
		EnvironmentType.LAUNDRIES_AND_WORKSHOPS,
		Light(15, LightType.LED),
		[Motor(horse_power = 5.0, efficiency = 0.92, name = "Batedor de argila")] +
		6 * [Motor(name = "Molde e cortador", horse_power = 1.0, power_factor = 0.85, efficiency = 0.95)]
	),

	Environment(
		"Produção 2",
		7.5,
		8,
		EnvironmentType.LAUNDRIES_AND_WORKSHOPS,
		Light(15, LightType.LED),
		2 * [Motor(horse_power = 2.0, power_factor = 0.8, efficiency = 0.95)] +
		[Motor(name = "Esteira Forno", horse_power = 2.0, power_factor = 0.83, efficiency = 0.93)] +
		[Device(name = "Forno", power = 6000, power_factor = 1.0)]
	),

	Environment(
		"Produção 3",
		4,
		10.4,
		EnvironmentType.LAUNDRIES_AND_WORKSHOPS,
		Light(15, LightType.LED),
		2 * [Motor(name = "Pintura", horse_power = 1.0, power_factor = 0.85, efficiency = 0.95)]
	),

	Environment(
		"Empacotamento e Carga e descarga",
		4,
		10.4,
		EnvironmentType.LAUNDRIES_AND_WORKSHOPS,
		Light(15, LightType.LED),
		2 * [Motor(name = "Esteiras", horse_power = 1.0, power_factor = 0.75, efficiency = 0.90)]
	),

]

installation = Installation(environments, InstallationType.THREE_PHASE)

print(installation)


Installation:
	Light Power: 4172.88W
	Light Number: 28
	Tug Power: 18700W
	Tug Number: 69
	Tue Power: 40266.0W
	Tue Number: 23
	Total Demanded Power: 21193.23W
	Installation Type: InstallationType.THREE_PHASE
	Phases: 3
	Total Demanded Apparent Power: 22308.66VA
	Total Demanded Current: 32.11A
	Input Branch Current: 40.14A
Enviroment:
	Name: Escritório 1 - Area: 8.97m² - Perimeter: 12.4m
	Light:
		Light: Lampada - 15 W - PF: 0.65 - 23.08 VA
		Recommended Power: 107.64W
		Minimum Light Power: 100W
		Light Power: 107.64W
		Minimum Light Points: 4
	Tug:
		Minimum Tug Number: 4
		Minimum Tug Power: 800W
	Tue:
		Minimum Tue Number: 1
Enviroment:
	Name: Escritório 2 - Area: 13.77m² - Perimeter: 15.6m
	Light:
		Light: Lampada - 15 W - PF: 0.65 - 23.08 VA
		Recommended Power: 165.24W
		Minimum Light Power: 160W
		Light Power: 165.24W
		Minimum Light Points: 4
	Tug:
		Minimum Tug Number: 5
		Minimum Tug Power: 1000W
	Tue:
		Minimum Tue Number: 1
Enviroment:
	Name: Recepção - Area: 14.28m² - Per

# Separação dos Circuitos

---

## Quadro Geral

- **Trifásico para monofásico**:
  - Geral quadro terminal 1: 50A (Monofásico) Fase 1
    - Disjuntor geral 50A (Monofásico)
    - Cabo de Geral até o quadro terminal 1: 10.0mm² (57A Monofásico)
  - Geral quadro terminal 2: 80A (Monofásico) Fase 2
    - Disjuntor geral 80A (Monofásico)
    - Cabo de Geral até o quadro terminal 2: 25.0mm² (101A Monofásico)
  - Geral quadro terminal 3: 80A (Monofásico) Fase 3
    - Disjuntor geral 80A (Monofásico)
    - Cabo de Geral até o quadro terminal 3: 25.0mm² (101A Monofásico)

- **Geral CCM1**: 50A (Trifásico)
    - Disjuntor geral 50A (Trifásico)
    - Cabo de Geral até o CCM1: 16.0mm² (68A Trifásico)
- **Geral CCM2**: 50A (Trifásico)
    - Disjuntor geral 50A (Trifásico)
    - Cabo de Geral até o CCM2: 16.0mm² (68A Trifásico)
- **Geral CCM3**: 50A (Trifásico)
    - Disjuntor geral 50A (Trifásico)
    - Cabo de Geral até o CCM3: 16.0mm² (68A Trifásico)

- **Geral Ramal 150A (Trifásico)**:
  - Cabo de Geral até o CCM1: 70.0mm² (171A Trifásico)
  - Por limitação do plano de distribuição da concessionária permitir uma potência máxima de 75kW, e pelo fato de que maior parte dos disjuntores, principalmente os dos CCMs serem extremamente sobre dimensionados devido aos requisitos para coordenação dos quadros segndo a norma, será utilizado um disjuntor geral de 150A (Trifásico), permitindo assim uma folga e coordenação dos disjuntores dos quadros que, na prática, não irão utilizar a potência máxima dimensionada.


## Quadro terminal 1

- **Número de disjuntores**: 17 (13 + 4 Reservas)
- **Número de TUEs**: 7
- **Poência demandada TUG + LUZ**: (930 + 1090 + 1060) = 3080W
- **Potência demandada TUE**: (2000 + 2000 + 2000 + 3000 + 3616) = 12616W
- **Potência demandada Total**: (3080 * 0.59) + (12616 * 0.6) = 9387W (considerando 9500W)
- **Potência aparante Total**: 9387 / 0.95 = 9881VA
- **Corrente Total**: 9881 / 220 = 44.91A
- **Corrente Geral**: 44.91 * 1.1 ~= 50A
- **Disjuntor no quadro geral (Pai)**: 50A (Monofásico)

### Escritório 1

- **Número de disjuntores**: 3
- **Número de TUEs**: 1
- **Poência demandada TUG + LUZ**: (130 + 800) = 930W
- **Potência demandada TUE**: 2000W

- **Disjuntor 1 (Circuito 1) (Iluminação)**: 15A (Monofásico)
  - Condutor: 1.5mm² (capacidade de 17.5A)
  - 4 pontos de iluminação
  - Potência requerida 100VA (considerando 200VA)
  - Potência ativa nominal: LED(FP: 0.65) 130W

- **Disjuntor 2 (Circuito 2) (Tomadas TUG)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 4 pontos de utilização
  - Potência requerida 800VA (considerando 1000VA)
  - Potência ativa nominal: TUG(FP: 0.8) 800W

- **Disjuntor 3 (Circuito 3) (Tomadas TUE)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 1 ponto de utilização (específico)
  - Potência requerida 2000VA (considerando 2500VA)
  - Potência ativa nominal: (FP/REND: 0.8) 2000W

### Escritório 2

- **Número de disjuntores**: 3
- **Número de TUEs**: 1
- **Poência demandada TUG + LUZ**: (130 + 960) = 1090W
- **Potência demandada TUE**: 2000W

- **Disjuntor 4 (Circuito 4) (Iluminação)**: 15A (Monofásico)
  - Condutor: 1.5mm² (capacidade de 17.5A)
  - 4 pontos de iluminação
  - Potência requerida 100VA (considerando 200VA)
  - Potência ativa nominal: LED(FP: 0.65) 130W

- **Disjuntor 5 (Circuito 5) (Tomadas TUG)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 5 pontos de utilização
  - Potência requerida 1000VA (considerando 1200VA)
  - Potência ativa nominal: TUG(FP: 0.8) 960W

- **Disjuntor 6 (Circuito 6) (Tomadas TUE)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 1 ponto de utilização (específico)
  - Potência requerida 2000VA (considerando 2500VA)
  - Potência ativa nominal: (FP/REND: 0.8) 2000W

### Recepção + Depósito

- **Número de disjuntores**: 3
- **Número de TUEs**: 1
- **Poência demandada TUG + LUZ**: (260 + 800) = 1060W
- **Potência demandada TUE**: 2000W

- **Disjuntor 7 (Circuito 7) (Iluminação)**: 15A (Monofásico)
  - Condutor: 1.5mm² (capacidade de 17.5A)
  - 2 pontos de iluminação na Recepção + 2 pontos no Depósito
  - Potência requerida 320VA (considerando 400VA)
  - Potência ativa nominal: LED(FP: 0.65) 260W

- **Disjuntor 8 (Circuito 8) (Tomadas TUG)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 5 pontos de utilização na Recepção + 2 pontos no Depósito
  - Potência requerida 700VA (considerando 1000VA)
  - Potência ativa nominal: TUG(FP: 0.8) 800W

- **Disjuntor 9 (Circuito 9) (Tomadas TUE - Recepção)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 1 ponto de utilização (específico)
  - Potência requerida 2000VA (considerando 2500VA)
  - Potência ativa nominal: (FP/REND: 0.8) 2000W

### Banho + Banheiro (TUE balanceamento de fase)

- **Número de disjuntores**: 2
- **Número de TUEs**: 2
- **Poência demandada TUG + LUZ**: 0W
- **Potência demandada TUE**: (1500 + 1500) = 3000W

- **Disjuntor 10 (Circuito 10) (Tomadas TUE Chapinha / Secador)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 1 ponto de utilização no Banheiro (específico)
  - Potência requerida 1200VA (considerando 1500VA)
  - Potência ativa nominal: (FP/REND: 1.0) 1500W

- **Disjuntor 11 (Circuito 11) (Tomadas TUE Chapinha / Secador)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 1 ponto de utilização no Banho (específico)
  - Potência requerida 1200VA (considerando 1500VA)
  - Potência ativa nominal: (FP/REND: 1.0) 1500W

### Cozinha + Refeitório (TUE balanceamento de fase)

- **Número de disjuntores**: 2
- **Número de TUEs**: 2
- **Poência demandada TUG + LUZ**: 0W
- **Potência demandada TUE**: (2116 + 1500) = 3616W

- **Disjuntor 12 (Circuito 12) (Tomadas TUE Refeitório Microondas)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 1 ponto de utilização na Cozinha (específico)
  - Potência requerida 2000VA (considerando 2300VA)
  - Potência ativa nominal: (FP/REND: 0.92) 2116W

- **Disjuntor 13 (Circuito 13) (Tomadas TUE Cozinha Chaleira)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 1 ponto de utilização no Refeitório (específico)
  - Potência requerida 1200VA (considerando 1500VA)
  - Potência ativa nominal: (FP/REND: 1.0) 1500W

---

## Quadro terminal 2

- **Número de disjuntores**: 11 (8 + 3 Reservas)
- **Número de TUEs**: 1
- **Poência demandada TUG + LUZ**: (2200 + 3140 + 930) = 6270W
- **Potência demandada TUE**: 7000W
- **Potência demandada Total**: (6270 * 0.4) + (7000 * 1.0) = 9509W (considerando 9600W)
- **Potência aparante Total**: 9509 / 0.95 = 10000VA
- **Corrente Total**: 10000 / 220 = 45.45A
- **Corrente Geral**: 45.45 * 1.1 ~= 50A
- **Disjuntor no quadro geral (Pai)**:
  - Necesário 50A (Monofásico)
  - Por possuir uma TUE com disjuntor de 35A, o disjuntor geral deve ser coordenado, assim utilizando um fator recomendado de 40% para a coordenação, o disjuntor geral do pai necessitaria de 87A para adaptar aos valores comerciais, será utilizado 45% que é permitido pela norma assim será usado um disjuntor de 80A (Monofásico)
  - **Disjuntor Geral**: 80A (Monofásico)

### Cozinha + Refeitório (Apenas TUGs)

- **Número de disjuntores**: 3
- **Número de TUEs**: 0
- **Poência demandada TUG + LUZ**: (200 + 1600 + 400) = 2200W
- **Potência demandada TUE**: 0W

- **Disjuntor 1 (Circuito 14) (Iluminação)**: 15A (Monofásico)
  - Condutor: 1.5mm² (capacidade de 17.5A)
  - 2 pontos de iluminação na Cozinha + 4 pontos no Refeitório
  - Potência requerida 200VA (considerando 300VA)
  - Potência ativa nominal: LED(FP: 0.65) 200W

- **Disjuntor 2 (Circuito 15) (Tomadas TUG Cozinha)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 3 pontos de utilização na Cozinha
  - Potência requerida 1800VA (considerando 2000VA)
  - Potência ativa nominal: TUG(FP: 0.8) 1600W

- **Disjuntor 3 (Circuito 16) (Tomadas TUG Refeitório)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)****
  - 4 pontos de utilização no Refeitório
  - Potência requerida 400VA (considerando 500VA)
  - Potência ativa nominal: TUG(FP: 0.8) 400W

### Vestiário + Banheiro + Banho

- **Número de disjuntores**: 3
- **Número de TUEs**: 1
- **Poência demandada TUG + LUZ**: (260 + 2880) = 3140W
- **Potência demandada TUE**: 7000W

- **Disjuntor 4 (Circuito 17) (Iluminação)**: 15A (Monofásico)
  - Condutor: 1.5mm² (capacidade de 17.5A)
  - 2 pontos de iluminação no Vestiário + 2 pontos no Banheiro + 2 ponto no Banho
  - Potência requerida 300VA (considerando 400VA)
  - Potência ativa nominal: LED(FP: 0.65) 260W

- **Disjuntor 5 (Circuito 18) (Tomadas TUG)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 2 pontos de utilização no Vestiário + 2 pontos no Banheiro + 2 ponto no Banho
  - Potência requerida 3600VA (considerando 3600VA)
  - Potência ativa nominal: TUG(FP: 0.8) 2880W

- **Disjuntor 6 (Circuito 19) (Tomadas TUE Chuveiro)**: 35A (Monofásico)
  - Condutor: 6.0mm² (capacidade de 41A)
  - 1 ponto de utilização no Banho (específico)
  - Potência requerida 7000VA (considerando 7000VA)
  - Potência ativa nominal: (FP/REND: 1.0) 7000W

### Supervisão de produção

- **Número de disjuntores**: 2
- **Número de TUEs**: 0
- **Poência demandada TUG + LUZ**: (130 + 800) = 930W
- **Potência demandada TUE**: 0W

- **Disjuntor 7 (Circuito 20) (Iluminação)**: 15A (Monofásico)
  - Condutor: 1.5mm² (capacidade de 17.5A)
  - 4 pontos de iluminação
  - Potência requerida 160VA (considerando 200VA)
  - Potência ativa nominal: LED(FP: 0.65) 130W

- **Disjuntor 8 (Circuito 21) (Tomadas TUG)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 4 pontos de utilização
  - Potência requerida 800VA (considerando 1000VA)
  - Potência ativa nominal: TUG(FP: 0.8) 800W

---

## Quadro terminal 3

- **Número de disjuntores**: 17 (13 + 4 Reservas)
- **Número de TUEs**: 1
- **Poência demandada TUG + LUZ**: (3224 + 3224 + 2848 + 2848) = 12144W
- **Potência demandada TUE**: 6500W
- **Potência demandada Total**: (12144 * 0.24) + (6500 * 1.0) = 9415W (considerando 9600W)
- **Potência aparante Total**: 9415 / 0.95 = 9910.52VA
- **Corrente Total**: 9910.52 / 220 = 45.04A
- **Corrente Geral**: 45.04 * 1.1 ~= 50A
- **Disjuntor no quadro geral (Pai)**:
  - Necesário 50A (Monofásico)
  - Por possuir uma TUE com disjuntor de 32A, o disjuntor geral deve ser coordenado, assim utilizando um fator recomendado de 40% para a coordenação, o disjuntor geral do pai necessita de 80A (Monofásico)
  - **Disjuntor Geral**: 80A (Monofásico)

### Produção 1

- **Número de disjuntores**: 3
- **Número de TUEs**: 0
- **Poência demandada TUG + LUZ**: (1064 + 1600 + 560) = 3224W
- **Potência demandada TUE**: 0W

- **Disjuntor 1 (Circuito 22) (Iluminação)**: 15A (Monofásico)
  - Condutor: 1.5mm² (capacidade de 17.5A)
  - 4 pontos de iluminação na Produção 1 (Cada ponto é uma luminária dupla)
  - Potência requerida 880VA (considerando 900VA)
  - Potência ativa nominal: FLUOR(fp: 0.5 / Preact: 4.8) 1064W

- **Disjuntor 2 (Circuito 23) (Tomadas TUG 1)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 3 pontos de utilização
  - Potência requerida 1800VA (considerando 2000VA)
  - Potência ativa nominal: TUG(FP: 0.8) 1600W

- **Disjuntor 3 (Circuito 24) (Tomadas TUG 2)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 6 pontos de utilização
  - Potência requerida 600VA (considerando 700VA)
  - Potência ativa nominal: TUG(FP: 0.8) 560W

### Produção 2

- **Número de disjuntores**: 4
- **Número de TUEs**: 1
- **Poência demandada TUG + LUZ**: (1064 + 1600 + 560) = 3224W
- **Potência demandada TUE**: 6500W

- **Disjuntor 4 (Circuito 25) (Iluminação)**: 15A (Monofásico)
  - Condutor: 1.5mm² (capacidade de 17.5A)
  - 4 pontos de iluminação na Produção 2 (Cada ponto é uma luminária dupla)
  - Potência requerida 880VA (considerando 900VA)
  - Potência ativa nominal: FLUOR(fp: 0.5 / Preact: 4.8) 1064W

- **Disjuntor 5 (Circuito 26) (Tomadas TUG 1)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 3 pontos de utilização
  - Potência requerida 1800VA (considerando 2000VA)
  - Potência ativa nominal: TUG(FP: 0.8) 1600W

- **Disjuntor 6 (Circuito 27) (Tomadas TUG 2)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 6 pontos de utilização
  - Potência requerida 600VA (considerando 700VA)
  - Potência ativa nominal: TUG(FP: 0.8) 560W

- **Disjuntor 7 (Circuito 28) (Tomadas TUE Forno)**: 32A (Monofásico)
  - Condutor: 6.0mm² (capacidade de 41A)
  - 1 ponto de utilização na Produção 2 (específico)
  - Potência requerida 6000VA (considerando 6500VA)
  - Potência ativa nominal: (FP/REND: 1.0) 6500W

### Produção 3

- **Número de disjuntores**: 3
- **Número de TUEs**: 0
- **Poência demandada TUG + LUZ**: (688 + 1600 + 560) = 2848W
- **Potência demandada TUE**: 0W

- **Disjuntor 8 (Circuito 29) (Iluminação)**: 15A (Monofásico)
  - Condutor: 1.5mm² (capacidade de 17.5A)
  - 4 pontos de iluminação na Produção 3 (Cada ponto é uma luminária dupla)
  - Potência requerida 580VA (considerando 600VA)
  - Potência ativa nominal: FLUOR(fp: 0.5 / Preact: 4.8) 688W

- **Disjuntor 9 (Circuito 30) (Tomadas TUG 1)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 3 pontos de utilização
  - Potência requerida 1800VA (considerando 2000VA)
  - Potência ativa nominal: TUG(FP: 0.8) 1600W

- **Disjuntor 10 (Circuito 31) (Tomadas TUG 2)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 6 pontos de utilização
  - Potência requerida 600VA (considerando 700VA)
  - Potência ativa nominal: TUG(FP: 0.8) 560W

### Empacotamento e Carga e descarga

- **Número de disjuntores**: 3
- **Número de TUEs**: 0
- **Poência demandada TUG + LUZ**: (688 + 1600 + 560) = 2848W
- **Potência demandada TUE**: 0W

- **Disjuntor 11 (Circuito 32) (Iluminação)**: 15A (Monofásico)
  - Condutor: 1.5mm² (capacidade de 17.5A)
  - 6 pontos de iluminação (Cada ponto é uma luminária dupla)
  - Potência requerida 580VA (considerando 600VA)
  - Potência ativa nominal: FLUOR(fp: 0.5 / Preact: 4.8) 688W

- **Disjuntor 12 (Circuito 33) (Tomadas TUG 1)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 3 pontos de utilização
  - Potência requerida 1800VA (considerando 2000VA)
  - Potência ativa nominal: TUG(FP: 0.8) 1600W

- **Disjuntor 13 (Circuito 34) (Tomadas TUG 2)**: 20A (Monofásico)
  - Condutor: 2.5mm² (capacidade de 24A)
  - 6 pontos de utilização
  - Potência requerida 600VA (considerando 700VA)
  - Potência ativa nominal: TUG(FP: 0.8) 560W

---

## CCM 1 - Produção 1

- **Tipo de motores**:
  - Motor tipo 1: 5CV (3.680kW) (Batedor e cilindrador de argila); rendimento 0.92; FP 0.85
    - Dmot1 = (5 * 736 * 0.83) / (0.92 * 0.85) = 3905.88VA
  - Motor tipo 2: 1CV (0.736kW) (Molde e cortador de cabo); rendimento 0.95; FP 0.85
    - Dmot2 = (1 * 736 * 0.7) / (0.95 * 0.85) = 638.01VA

- **Número de disjuntores**: 10 (7 + 3 Reservas)
- **Número de motores controlados**:
  - Motor tipo 1: 1
  - Motor tipo 2: 6
  - **Total de motores**: 7
- **Potência demandada**:
  - Motores tipo 1: (1 * 3905.88 * 1.0) = 3905VA
  - Motores tipo 2: (6 * 638.01 * 0.75) = 2871VA
  - **Potência demandada Total**: 6776.88VA
- **Potência aparante Total**: 6776.88VA
- **Corrente Total**: 6776.88 / (220 * 3) = 10.27A
- **Corrente Geral**: 10.27A * 1.2 ~= 12.34A
- **Disjuntor no quadro geral (Pai)**:
  - Necesário 12.34A (Trifásico)
  - Por possuir um disjuntor de 25A no circuito 35, o disjuntor geral deve ser coordenado, assim utilizando um fator recomendado de 45% para a coordenação, o disjuntor geral do pai necessita de 55A (Trifásico)
  - **Disjuntor Geral**: 60A (Monofásico)

### Motores tipo 1

- **Disjuntor 1 (Circuito 35)**: 25A (Trifásico)
  - Condutor: 4.0mm² (capacidade de 28A)
  - Potência requerida 3905.88VA

### Motores tipo 2

- **Disjuntor 2 (Circuito 36)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 638.01VA

- **Disjuntor 3 (Circuito 37)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 638.01VA

- **Disjuntor 4 (Circuito 38)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 638.01VA

- **Disjuntor 5 (Circuito 39)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 638.01VA

- **Disjuntor 6 (Circuito 40)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 638.01VA

- **Disjuntor 7 (Circuito 41)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 638.01VA

---

## CCM 2 - Produção 2  + Produção 3

- **Tipo de motores**:
  - Motor tipo 1: 2CV (1.472kW) (Rotacão de canecas dentro do forno); rendimento 0.95; FP 0.8
    - Dmot1 = (2 * 736 * 0.7) / (0.95 * 0.8) = 1355.79VA
  - Motor tipo 2: 2CV (1.472kW) (Moldador do corpo); rendimento 0.93; FP 0.83
    - Dmot2 = (2 * 736 * 0.7) / (0.93 * 0.83) = 1334.88VA
  - Motor tipo 3: 1CV (0.736kW) (Compressor pintura); rendimento 0.95; FP 0.85
    - Dmot3 = (1 * 736 * 0.7) / (0.95 * 0.85) = 638.01VA

- **Número de disjuntores**: 8 (5 + 3 Reservas)
- **Número de motores controlados**:
  - Motor tipo 1: 2
  - Motor tipo 2: 1
  - Motor tipo 3: 2
  - **Total de motores**: 5
- **Potência demandada**:
  - Motores tipo 1: (2 * 1355.79 * 0.85) = 2304.84VA
  - Motores tipo 2: (1 * 1334.88 * 1) = 1334.88VA
  - Motores tipo 2: (2 * 638.01 * 0.85) = 1084.61VA
  - **Potência demandada Total**: 4724.33VA
- **Potência aparante Total**: 4724.33VA
- **Corrente Total**: 4724.33 / (220 * 3) = 7.18A
- **Corrente Geral**: 7.18A * 1.2 ~= 8.7A
- **Disjuntor no quadro geral (Pai)**:
  - Necesário 8.7A (Trifásico)
  - Por possuir disjuntores de 20A, o disjuntor geral deve ser coordenado, assim utilizando um fator recomendado de 45% para a coordenação, o disjuntor geral do pai necessita de 50A (Trifásico)
  - **Disjuntor Geral**: 50A (Trifásico)

### Motores tipo 1

- **Disjuntor 1 (Circuito 42)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 1355.79VA

- **Disjuntor 2 (Circuito 43)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 1355.79VA

### Motores tipo 2

- **Disjuntor 3 (Circuito 44)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 1334.88VA

### Motores tipo 3

- **Disjuntor 4 (Circuito 45)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 638.01VA

- **Disjuntor 5 (Circuito 46)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 638.01VA

---

## CCM 3 - Empacotamento e Carga e descarga

- **Tipo de motores**:
  - Motor tipo 1: 1CV (0.736kW) (Esteira e tanque); rendimento 0.9; FP 0.75
    - Dmot1 = (1 * 736 * 0.7) / (0.9 * 0.75) = 763.26VA

- **Número de disjuntores**: 5 (2 + 3 Reservas)
- **Número de motores controlados**:
  - Motor tipo 1: 2
  - **Total de motores**: 2
- **Potência demandada**:
  - Motores tipo 1: (2 * 763.26 * 0.85) = 1297.54VA
  - **Potência demandada Total**: 1297.54VA
- **Potência aparante Total**: 1297.54VA
- **Corrente Total**: 4724.33 / (220 * 3) = 1.96A
- **Corrente Geral**: 1.96 * 1.2 ~= 2.35A
- **Disjuntor no quadro geral (Pai)**:
  - Necesário 2.35A (Trifásico)
  - Por possuir disjuntores de 20A, o disjuntor geral deve ser coordenado, assim utilizando um fator recomendado de 45% para a coordenação, o disjuntor geral do pai necessita de 50A (Trifásico)
  - **Disjuntor Geral**: 50A (Monofásico)

### Motores tipo 1

- **Disjuntor 1 (Circuito 47)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 763.26VA

- **Disjuntor 2 (Circuito 48)**: 20A (Trifásico)
  - Condutor: 2.5mm² (capacidade de 21A)
  - Potência requerida 763.26VA
